iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/execution/config/metadata/ShowFunctionsTask.java

```diff
@@ -55,9 +61,29 @@
 
 public class ShowFunctionsTask implements IConfigTask {
```

Comment:
will this class be used for table model? it seems that you don't deal with that

Reply:
CREATE/DROP/SHOW FUNCTION statement of table model will be supported in the next PR. This PR just want to refactor common codes about UDF management and make sure it won't affect tree model.